data-generation/src/main/scala/thesis/Landy.scala

```diff
@@ -33,4 +36,67 @@ class Landy(val graph: LandyAttributeGraph) extends TemporalGraph[LandyVertexPay
   }
```

Comment:
Jeg klarer ikke helt å kommentere på riktig linje, men jeg tror `this.` som et prefix for klassevariable(?) er valgfritt. Er vel litt smaksak om man vil gjøre det tydelig hvor denne variabelen kommer fra